app/models.py

```diff
@@ -43,30 +45,34 @@ class Categories(db.Model):
     __tablename__ = 'categories'
 
     id = db.Column(db.Integer, primary_key=True)
-    category_name = db.Column(db.String(100), unique=True)
-    category_owner_email = db.Column(db.String(100))
-    user_id = db.Column(db.Integer, db.ForeignKey('users.id'), nullable=False)
+    category_name = db.Column(db.String(100))
+    users_id = db.Column(db.Integer, db.ForeignKey(Users.id), nullable=False)
     created_at = db.Column(db.DateTime, default=db.func.current_timestamp())
     updated_at = db.Column(db.DateTime, default=db.func.current_timestamp(),
                            onupdate=db.func.current_timestamp())
-    recipes = db.relationship(
-        'Recipes', order_by='recipes.id', cascade="all, delete-orphan")
+    recipes = db.relationship('Recipes', order_by='Recipes.id', cascade="all, delete-orphan")
 
-    def __init__(self, category_name):
-        """initialize with user email."""
+    def __init__(self, category_name, users_id):
+        """initialize with category name and user id."""
         self.category_name = category_name
+        self.users_id = users_id
 
     def save(self):
         db.session.add(self)
         db.session.commit()
 
+    @staticmethod
+    def name_unique(owner_id, category_name):
+        check_category_existence = Categories.query.filter_by(category_name=category_name, users_id=owner_id).first()
```

Comment:
Are you using a python linter? This line looks too long.

Reply:
Done